README.md

```diff
@@ -168,6 +170,22 @@ Success!
 Your keys can be found at: <YOUR_FOLDER_PATH>
 ```
 
+###### `generate-bls-to-execution-change` Arguments 
+
+You can use `bls-to-execution-change --help` to see all arguments. Note that if there are missing arguments that the CLI needs, it will ask you for them.
+
+| Argument | Type | Description |
+| -------- | -------- | -------- |
+| `--bls_to_execution_changes_folder` | String. Pointing to `./bls_to_execution_changes` by default | The folder path for the `bls_to_execution_change-*` JSON file(s) |
+| `--chain` | String. `mainnet` by default | The chain setting for the signing domain. |
+| `--mnemonic` | String. mnemonic split by space.  | The mnemonic you used to create withdrawal credentials. |
+| `--mnemonic_password` | Optional string. Empty by default. | The mnemonic password you used in your key generation. Note: It's not the keystore password. |
+| `--validator_start_index` | Non-negative integer | The index of the first validator's keys you generated withdrawal credentials with the mnemonic. |
```

Comment:
"The index of the first validator's keys..." does this mean I cannot start at a random index?
Example: I want to change/set the eth1 address for only one validator with the index 50.
Could I just do  --validator_start_index 50 --validator_indices some_beacon_index_of_that_val ?
Do I really need to start with the first validator I created?

Reply:
This would also imply that I have to do everything in one go?
What if I wanted to have two different withdrawal addresses?
Example: 

- index 0-50 withdrawal_address_1 and 
- index 51-100 withdrawal_address_2
How could I do this then if it always expects the index 0? 
On another note, I didn't see a validation requiring this field to be a 0, thus it might work with a non-0 integer?